docs/install_guides/installing-openvino-linux.md

```diff
@@ -151,23 +151,20 @@ The environment variables are set. Next, you can download some additional tools.
 ## <a name="optional-steps"></a>Step 5 (Optional): Configure Inference on Non-CPU Devices
 
 @sphinxdirective
-.. tab:: GNA
-
-   Only if you want to enable the toolkit components to use Intel® Gaussian & Neural Accelerator (GNA) on your system, follow the steps in :ref:`GNA Setup Guide <gna guide>`.
```

Comment:
why have we removed GNA? The driver for GNA needs to be installed

Reply:
It's removed by mistake. Added it back.